pkg/velaql/providers/query/types/type.go

```diff
@@ -65,6 +65,9 @@ func (s *ServiceEndpoint) String() string {
 	if protocol == "tcp" {
 		return fmt.Sprintf("%s:%d%s", s.Endpoint.Host, s.Endpoint.Port, path)
 	}
+	if s.Endpoint.Port == 0 {
```

Comment:
what does it mean when port is 0?

Reply:
I'm not sure if loadbalancer provided by cloud vendor will fill the `port.nodePort` with 80. If all vendor will fill that field, this change can be rollback. Basically this is just a guard.